src/main/java/org/apache/sling/feature/cpconverter/cli/ContentPackage2FeatureModelConverterLauncher.java

```diff
@@ -133,6 +135,9 @@
     @Option(names = { "-u", "--unreferenced-artifacts-output-directory" }, description = "The output directory where unreferenced artifacts will be deployed.", required = false)
     private File unreferencedArtifactsOutputDirectory;
 
+    @Option(names = { "-c", "--cndUri" }, description = "A URI returning a CND file. Used for resolving namespace prefixes in Sling-Initial-Content in addition to the namespaces given in the bundle headers", required = false)
+    private List<URI> cndUris = new ArrayList<>();
```

Comment:
So, basically, use the --seed-feature (if any), parse the repoinit (if any), and use the given namespaces there (if any)...

Reply:
Do you have some pointers how to easily extract the CNDs from there?

Reply:
I was hoping you could do something similar like we did for the service users - i.e., parse the repoinit and use the information from it. An example would be here: https://github.com/apache/sling-org-apache-sling-feature-cpconverter/blob/5be6cf3b0ce39372598343f015c9276cdb220909/src/main/java/org/apache/sling/feature/cpconverter/accesscontrol/DefaultAclManager.java#L185

Reply:
Done in https://github.com/apache/sling-org-apache-sling-feature-cpconverter/pull/74/commits/36340ec7927e6679e831fc26a700760033e3500b. Thanks for the pointer.

Reply:
Looks good - thanks